### ISSUE TITLE
why does Checkbox need a separate listener for keyboard input?

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/scenery-phet#480]
Noted while working on https://github.com/phetsims/scenery-phet/issues/479.

Checkbox has this code related to "firing" of its state change:

```js
    // interactivity
    var fire = function() {
      if ( self.enabledProperty.value ) {
        var newValue = !property.value;
        toggledEmitter.emit( newValue );
      }
    };

    var checkboxButtonListener = new ButtonListener( {
      fire: fire
    } );
    this.addInputListener( checkboxButtonListener );

    // a11y - fire the listener when checkbox is clicked with keyboard or assistive technology
    var changeListener = {
      change: fire
    };
    this.addInputListener( changeListener );
```

Why do we need to add another listener for a11y that calls the exact same function as the ButtonListener that was already added?  This should be handled by ButtonListener; it should be able to handle _all types_ of button input, not just pointer.

**Comment by pixelzoom** [comment]
Transferring this issue from scenery-phet to sun, since Checkbox lives in sun.
